en/developer_guide/routing.rst

```diff
@@ -0,0 +1,79 @@
+Routing
+=======
+
+Base URL
+--------
+
+The page base url is compose with domain name and language prefix.
```

Comment:
In Open Orchestra, a page url is composed by the domain, a language prefix and the page path